unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -53,6 +53,15 @@
 MAX_CONTENT_PARAM_BYTE_SIZE = 8000
 
 
+class FileDataMetadata(BaseModel):
+    collection_name: str
+    keyspace: Optional[str] = None
+
+
+class AstraDBBatchFileData(BatchFileData):
+    additional_metadata: FileDataMetadata
```

Comment:
I think the `FileDataMetadata` is fine, harder to mistake then when referring to a dict but

1. Name's a little confusing, we have both metadata and additional metadata for FileData and this is named like it should be the first one in my opinion
2. I don't like this subclass, we can just use `model_validate` and `model_dump` when we want to operate on `additional_metadata`. This completely changes that field's type and right now our typing doesn't even catch that in the code below.